plugins/modules/aci_rest.py

```diff
@@ -453,6 +470,10 @@ def main():
     # NOTE By setting aci.path we ensure that Ansible displays accurate URL info when the plugin and the aci_rest module are used.
     aci.path = path.lstrip("/")
     aci.url = "{0}/{1}".format(aci.base_url, aci.path)
+
+    if aci.params.get("method") == "get" and page_size:
+        aci.path = "{0}?page={1}&page-size={2}".format(aci.path, page, page_size)
+        aci.url = update_qsl(aci.url, {"page": page, "page-size": page_size})
```

Comment:
There seems to be a issue when doing setting url and path in this way when path contains other filters. As an example if path is "/api/class/fvSubnet.json?order-by=fvSubnet.descr" and you would add page information as above it would display the url as `"https://173.36.219.82/api/class/fvSubnet.json?order-by=fvSubnet.descr?page=0&page-size=3"` but would sent correctly to `https://173.36.219.82:443/api/class/fvSubnet.json?order-by=fvSubnet.descr&page=0&page-size=3`

as temp workaround tested that leveraging the update_qsl() function will display it correctly

```suggestion
        aci.path = update_qsl(aci.path, {"page": page, "page-size": page_size})
        aci.url = update_qsl(aci.url, {"page": page, "page-size": page_size})
```

I do think however we should rootcause this, because it is likely that similar behaviour is present for rsp-subtree

Reply:
I´ll look into this. thanks.